patterns/multi-tenancy-with-teams/main.tf

```diff
@@ -50,10 +50,10 @@ locals {
 
 module "eks" {
   source  = "terraform-aws-modules/eks/aws"
-  version = "~> 19.16"
+  version = "~> 19.21"
```

Comment:
need to update teams module to add native access entry support and then we can remove the `aws-auth` bits here and update to v20